src/sentry/conf/server.py

```diff
@@ -1689,6 +1689,17 @@ def get_sentry_sdk_config():
     KAFKA_INGEST_TRANSACTIONS: {"cluster": "default", "topic": KAFKA_INGEST_TRANSACTIONS},
 }
 
+SENTRY_EVENTSTREAM = "sentry.eventstream.snuba.SnubaEventStream"
+SENTRY_EVENTSTREAM_OPTIONS = {
+    "kafka": {
```

Comment:
@tkaemming , Not sure if this is what you meant when suggesting to move the config in the eventstream options. Or did you mean to use the ServiceDelegator? IT seems this should not be the case since we statically pick one of the two backends by config.